app/views/payment_gateways/_stripe.html.erb

```diff
@@ -10,119 +10,123 @@
     <br/>
   <% end %>
 
-  <%= semantic_form_for 'stripe',
-                        url: hosted_success_admin_account_stripe_path,
-                        method: 'POST',
-                        html: { id: 'stripe-form',
-                                class: "#{current_account.credit_card_stored? ? 'hidden' : 'card-missing'} form-horizontal",
-                                data: { stripe_publishable_key: @stripe_publishable_key, secret: @intent.client_secret }
-                        } do |form| %>
-
-    <fieldset>
-      <legend>Credit card details</legend>
-      <div id="card-element" class="col-md-10">
-        <!-- a Stripe Element will be inserted here. -->
-      </div>
-
-      <!-- Used to display form errors -->
-      <div id="card-errors" role="alert"></div>
-
-      <%= render partial: 'payment_gateways/stripe_billing_address', object: current_account.billing_address %>
-
-      <%= form.hidden_field :payment_method_id %>
-    </fieldset>
-
-    <fieldset>
-      <div class="form-group">
-        <div class="col-md-10 operations">
-          <%= form.submit 'Save details', id: 'stripe-submit',
-                                          class: 'btn btn-primary pull-right',
-                                          disabled: true %>
+  <% if @intent.nil? %>
+    <p>Stripe is unavailable at the moment. Please try again later.</p>
```

Comment:
We should figure out in which scenario this is possible, but I doubt that the reason is Stripe being unavailable... in that case, an error would be raised and we are not catching it at the moment so it would just crash (5XX error)

Reply:
> We should figure out in which scenario this is possible

I mean figuring out the root cause. This does not seem like a Frontend problem.

Reply:
Yes, it's not a real error message.. but I thing this one might be a random error. Related to timing maybe, weird user behavior, maybe mixed with a slow connection... I'll try a couple more scenarios in case I find out something else.